managedkafka/clusters/create_cluster.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clusters
+
+// [START managedkafka_create_cluster]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/managedkafka/apiv1/managedkafkapb"
+	"google.golang.org/api/option"
+
+	managedkafka "cloud.google.com/go/managedkafka/apiv1"
+)
+
+func createCluster(w io.Writer, projectID, region, clusterID, subnet string, cpu, memoryBytes int64, opts ...option.ClientOption) error {
+	// projectID := "my-project-id"
+	// region := "us-central1"
+	// clusterID := "my-cluster"
+	// subnet := "my-subnet"
+	// cpu := 3
+	// memoryBytes := 3221225472
```

Comment:
Why this number? Is this a sensible amount for the number of CPUs?

Reply:
The API at the moment only take memory in bytes.

It is required that the CPU:Memory ratio is between 1:1GiB and 1:8GiB. In this example with CPU being at 3, memory has to be 3 GiB (3221225472 bytes).

Reply:
I think it might be good to add this as a comment to the sample. Probably good as a comment just before defining `capacityConfig`.